ynr/apps/uk_results/templates/uk_results/ballot_paper_results_form.html

```diff
@@ -3,6 +3,8 @@
 {% load parse_iso %}
 
 {% block hero %}
+<script type='text/html' src="/static/js/results.js"></script>    
```

Comment:
Because you've added this to `ynr/settings/base.py`, you don't also need it here - the file will get concatenated in to the main JS file